src/themes/grayscale.js

```diff
@@ -0,0 +1,145 @@
+export default {
+  line: {
+    data: {
+      stroke: "#252525",
+      strokeWidth: 2,
+      fill: "none",
+      opacity: 1
+    },
+    labels: {
+      padding: 5,
+      fontFamily: "Poppins, Helvetica-Neue, sans-serif",
```

Comment:
Oh! I forgot to give you a more robust font stack: 
`"Poppins, 'Century Gothic', 'Trebuchet MS', sans-serif"`